src/main.js

```diff
@@ -24,4 +24,59 @@ for ( let k = 0; k < arr1.length; k++) {
     if (arr1[k] % 2 === 0) {
         console.log(arr1[k]);
     }
-}
\ No newline at end of file
+}
+function palindrome(str){
+    str = str.toLowerCase().replace(/[^а-яa-z1-9]/gi, '');
+    const lastIndex = str.length - 1;
+    for (let i = 0; i < str.length/2; i++) {
+        if (str[i] !== str[lastIndex - i]) {
+            return false;
+        }
+    }
+    return true;
+}
+console.log(palindrome("шалаш"));
+console.log(palindrome("ротор"));
+console.log(palindrome("машина"));
+
+/*function min(a,b) {
+    if (a < b) {
+        return a;
+    }
+    return b;
+}
+*/
+function min(a,b) {
+    return a < b ? a : b;
+}
+
+console.log(min(3,5));
+console.log(min(6,4));
+
+/*function max(a,y) {
+    if (a > y) {
+        return a;
+    }
+    return y;
+}
+ */
+function max(a,y) {
+    return a > y ? a : y;
+}
+console.log(max(4,8));
+console.log(max(10,2));
+
+let randomArray = [];
+for (let i = 0; i < 10; i++) {
+    randomArray.push(Math.round(Math.random() * 100));
+}
+console.log(randomArray);
+
+function replace(number) {
+    if (`${number}`.includes("0")) {
+        return `${number}`.replaceAll("0", "zero");
+    } else {
+        return number;
+    }
+}
+console.log(replace(randomArray));
```

Comment:
According to the homework your function must return an array of numbers and strings [12, 53, '2zero', 18, 22, '1zerozero', 43, 57, '5zero', 1]. The function returns just a string